src/main/controllers/AddressDetailsController.ts

```diff
@@ -14,7 +14,7 @@ export default class AddressDetailsController {
   private readonly addressDetailsContent: FormContent = {
     fields: {
       address1: {
-        id: 'address-line1',
+        id: 'address1',
```

Comment:
Why have you renamed all of these?

Reply:
The address details page has the postcode look up API part and the manual entry part. There is a function which picks up these ids to display or hide.

Reply:
https://github.com/hmcts/et-sya-frontend/blob/master/src/main/assets/js/address/links.ts